aerolyzer/aerosol.py

```diff
@@ -0,0 +1,81 @@
+'''
+Aerosol Analysis Class
+Description: This file contains the AeroData class which takes an image and analyses the colors present.
+'''
+import math
+import scipy.stats as stats
+import retrieve_image_data
```

Comment:
??? Isn't retrieve_image_data a class in the aerolyzer library? If so, then this import is incorrect. If not, then what library is this?